python/get_config.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python -B
+
+# Insert the parent directory into the module import search path.
+import os
+import sys
+sys.path.insert(1, os.path.dirname(sys.path[0]))
```

Comment:
This shouldn't be necessary, as you are already in the "python" directory and the current directory is always added to the path